app/views/support_interface/import_references/index.html.erb

```diff
@@ -0,0 +1,39 @@
+<% content_for :title, 'Import references' %>
+<% content_for :before_content, govuk_back_link_to(support_interface_applications_path) %>
+
+<% if @error %>
+  <div class="govuk-error-summary" aria-labelledby="error-summary-title" role="alert" tabindex="-1" data-module="govuk-error-summary">
+    <h2 class="govuk-error-summary__title" id="error-summary-title">
+      There is a problem
+    </h2>
+    <div class="govuk-error-summary__body">
+      <ul class="govuk-list govuk-error-summary__list">
+        <li>
+          <a href="#csv_file"><%= @error %></a>
+        </li>
+      </ul>
+    </div>
+  </div>
+<% end %>
+
+<h1 class='govuk-heading-xl'>Import references</h1>
+
+<%= form_tag support_interface_import_references_path, method: :post, multipart: true do %>
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+      <p class="govuk-body">
+        You’ll need to <a href="https://docs.google.com/spreadsheets/d/18Lr1SWD20o_sPbyj7gsTwkaJO9jwL-YZTTdQfQOuIiw/export?format=csv" class="govuk-link" rel="download">download a CSV of the latest reference responses</a> from Google Docs before you can upload it here.</p>
+
```

Comment:
Perhaps we explain to the user here what the importing of the CSV will do? Something that says that we'll add feedback to the references, change the state, etc etc.